docs/administration.md

```diff
@@ -97,3 +97,58 @@ Note: This will only work when the replication factor is >= 2
 - Modify the cluster config file to replace the previous node address with the new node address.
 - Restart the cluster
 - Wait for the 1st sync (10 minutes) to validate Index connections
+
+#### Metrics
+
+Pilosa can be configured to emit metrics on its internal processes in 2 formats, Expvar or StatsD.  Metric recording is disabled by default and enabled through the configuration file.  
+Through the configuration file choose the metric type, host to receive events, and polling interval for runtime metrics.
+
+##### Tags
+StatsD Tags are in DataDog format (key:value), and we tag the following:
```

Comment:
can we say `adhere to the` instead of `are in`?
also, can we link to those DataDog format specs on the DataDog site somewhere?